vmdb/product/toolbars/conditions_center_tb.yaml

```diff
@@ -13,5 +13,5 @@
     :items:
     - :button: condition_new
       :image: new
-      :text: 'Add a New #{ui_lookup(:model=>@sb[:folder])} Condition'
-      :title: 'Add a New #{ui_lookup(:model=>@sb[:folder])} Condition'
\ No newline at end of file
+      :text: 'Add a New #{@sb[:folder].upcase == "VM" ? ui_lookup(:model=>@sb[:folder]).upcase : ui_lookup(:model=>@sb[:folder])} Condition'
```

Comment:
@h-kataria Shouldn't ui_lookup work fine here and return us the properly cased "VM" for the model?  Could this be a bug somewhere in ui_lookup or the methods/constants it uses?